apps/studio/src/features/dashboard/SiteList.tsx

```diff
@@ -115,13 +116,47 @@ const SuspendableSiteList = (): JSX.Element => {
       <Flex flexDirection="column" gap="2rem">
         <SimpleGrid columns={3} gap="2.5rem" width="100%">
           {sites.map((site) => (
-            <Card key={site.id} width="100%">
-              <CardHeader>
-                <Link href={`/sites/${site.id}`} as={NextLink}>
-                  {site.name}
-                </Link>
-              </CardHeader>
-            </Card>
+            <Link
+              href={`/sites/${site.id}`}
+              as={NextLink}
+              textDecoration="none"
+              color="links.neutral-hover"
+              _hover={{
+                textDecoration: "none",
+                color: "links.neutral-hover",
+              }}
+            >
+              <Flex
+                key={site.id}
+                flexDirection="column"
+                gap="1rem"
+                width="100%"
+              >
+                <Image
+                  src={site.config.logoUrl}
+                  alt={site.name}
+                  borderRadius="0.5rem"
+                  border="1.5px solid #EDEDED"
+                  width="100%"
+                  height="100%"
+                  objectFit="cover"
+                  aspectRatio="1/1"
+                  backgroundColor="white"
+                  fallbackSrc="/isomer-sites-placeholder.png"
+                />
+                <Flex flexDirection="column" gap="0.5rem">
+                  <Text
```

Comment:
nit: i think this renders as `p` but we can use `as` to render a proper header! (this is a nit so feel free to ignore HAHAHAH)

Reply:
oh as in using `h6` as per figma? sounds good, lemme do that, thanks!